task-2/src/store/thunk/index.js

```diff
@@ -0,0 +1,9 @@
+// export const allLoadedMovies = () => async (dispatch) => {
+// 	const apiLink = 'http://localhost:4000/movies?limit=10';
```

Comment:
It's a good start.
Just don't forget to change it according to the above comment.
Request making logic in the "src/core/api" as well as the related constants (url divided into a baseUrl part and the specific endpointUrl part.
In the thunk itself should just be an async call to some function and a dispatch call with the awaited result from that function.

Reply:
That's all part of "separation of concerns" principle.
Thunks are concerned with dispatching async events.
Api is concerned of making requests to specific urls.